arcade/examples/platform_tutorial/06_camera.py

```diff
@@ -121,19 +121,17 @@ def on_key_release(self, key, modifiers):
             self.player_sprite.change_x = 0
 
     def center_camera_to_player(self):
-        screen_center_x = self.player_sprite.center_x - (self.camera.viewport_width / 2)
-        screen_center_y = self.player_sprite.center_y - (
-            self.camera.viewport_height / 2
-        )
+        screen_center_x = self.player_sprite.center_x
+        screen_center_y = self.player_sprite.center_y
 
         # Don't let camera travel past 0
-        if screen_center_x < 0:
-            screen_center_x = 0
-        if screen_center_y < 0:
-            screen_center_y = 0
+        if screen_center_x - self.width/2 < 0:
+            screen_center_x = self.width/2
+        if screen_center_y - self.height/2 < 0:
+            screen_center_y = self.height/2
         player_centered = screen_center_x, screen_center_y
 
-        self.camera.move_to(player_centered)
+        arcade.camera.controllers.simple_follow_2D(0.2, player_centered, self.camera.view_data)
```

Comment:
Probably should use a named parameter or variable so that 0.2 isn't a random number appearing out of no where. I think elsewhere you used a camera_panning variable of some sort.